packages/cypher-builder/src/clauses/Call.test.ts

```diff
@@ -122,4 +122,31 @@ describe("CypherBuilder Call", () => {
             }
         `);
     });
+
+    it("CALL with unwind", () => {
+        const node = new Cypher.Node({ labels: ["Movie"] });
+
+        const matchClause = new Cypher.Match(node)
+            .where(Cypher.eq(new Cypher.Param("aa"), new Cypher.Param("bb")))
+            .return([node.property("title"), "movie"]);
+
+        const clause = new Cypher.Call(matchClause).unwind("*");
+        console.log(clause);
+        const queryResult = clause.build();
+        expect(queryResult.cypher).toMatchInlineSnapshot(`
+            "CALL {
+                MATCH (this0:\`Movie\`)
+                WHERE $param0 = $param1
+                RETURN this0.title AS movie
+            }
+            UNWIND *"
```

Comment:
Do you not always need to alias an `UNWIND`?

Reply:
I think it was more of a general observation of, should it be possible to generate invalid Cypher like this through the Cypher Builder?